src/components/rounds-list/rounds-list.tsx

```diff
@@ -0,0 +1,24 @@
+import { Round } from "@/app/events/[event]/participants/page";
+import { ParticipantsList } from "../participants-list";
+
+interface RoundsListProps {
```

Comment:
Same as the others, `interface` -> `type`.

Reply:
Done